Add navigation to actor details in CastAndCrew

tomecky1 opened the pull request:
Created a reusable `useNavigation` hook for handling navigations. Integrated it into `CastAndCrew` to enable navigation to actor detail pages on item click. Enhanced UI with pointer cursor for clickable elements.

tomecky1 committed:
Add navigation to actor details in CastAndCrew

Created a reusable `useNavigation` hook for handling navigations. Integrated it into `CastAndCrew` to enable navigation to actor detail pages on item click. Enhanced UI with pointer cursor for clickable elements.

albert383 commented on src/features/CastAndCrew/index.js:
```diff
@@ -1,27 +1,14 @@
-  // useEffect(() => {
-  //   dispatch(fetchMovies());
-  // }, [dispatch]);
-
```
If its not needed its better to remove commented area

tomecky1 replied:
I will fix it

tomecky1 replied:
Done

albert383 reviewed (approved):
Looks good

tomecky1 committed:
Simplify comments and remove translation remnants.

Updated comments to use consistent English phrasing and cleaned up unnecessary translated notes. No functional changes were made to the application logic.
